package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "community",
+  "version": "0.0.1",
+  "description": "A collection of community specific resources, such as rules, COC, and team services",
+  "main": "index.js",
+  "scripts": {
+    "doctoc": "node_modules/.bin/doctoc ."
```

Comment:
You shouldn't actually need the `node_modules/.bin` prefix for anything in the `scripts` section, NPM automatically adds that to the `PATH` when running `npm run <script_target>` (see [here](https://docs.npmjs.com/misc/scripts#path))

Reply:
oh i didn't realize that. I'll fix 'er up.